utils/Party.ts

```diff
@@ -15,11 +15,32 @@ const partyNameFromSlug = (slug: string) => {
     case "Reform":
       return "Reform UK";
     default:
-      "Other";
+      return "Other";
   }
 };
 
-const partyCssClassFromSlug = (slug: string) => {
+const partyColorFromSlug = (slug: PartySlug) => {
```

Comment:
It might be worth us defining in our SCSS file specific color variables for each party, then if they change, we just have one place to update rather than the SCSS _and_ here. :thinking: